package.json

```
@@ -21,7 +21,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "axios": "^1.6.4",
+    "axios": "^1.7.9",
     "commitlint": "^17.6.3",
```

```suggestion
    "commitlint": "^19.5.0",
```
can you handle this one too ? https://app.snyk.io/org/rudder-integrations/project/01929853-9c91-4670-971b-18e4220aea4a#issue-SNYK-JS-CROSSSPAWN-8303230